frontend/test/metabase/scenarios/custom-column/cc-expression-editor.cy.spec.js

```diff
@@ -49,7 +49,7 @@ describe("scenarios > question > custom column > expression editor", () => {
       .type("{movetoend}{backspace}", { force: true })
       .blur();
 
-    cy.findByText("Unexpected end of input");
+    cy.findByText("Expected expression");
```

Comment:
IMO either message is confusing for the end user. Perhaps something like "Incomplete expression"?

Reply:
I think we should tackle this in a separate PR along with i8n - with error markers I think we'll discover a few messages that don't quite make sense with the extra information